e2e/paths/caseSubmitted_test.js

```diff
@@ -1,11 +1,16 @@
+/* global locate*/
 const config = require('../config.js');
 const fields = {
   documentLink: 'ccd-read-document-field>a',
 };
+const monthNames = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun',
```

Comment:
I saw monthNames in another place also. If they were in a fixture there would be no duplication

Reply:
👍